README.aspnetcore.md

```diff
@@ -15,10 +15,14 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build-nightly`
 - `1.1.2`, `1.1`, `1`
     - [`1.1.2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/runtime/Dockerfile)
     - [`1.1.2-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/nanoserver/runtime/Dockerfile)
-- `2.0.0`, `2.0`, `2`, `latest`
+- `2.0.0`, `2.0`
```

Comment:
It's a good start. I'm going to add a script to the repo to help me remember all those variables required to generate.

Also, it appears the manifest file in my repo needs updating....

```
READING MANIFEST
----------------
Newtonsoft.Json.JsonSerializationException: Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'System.Collections.Generic.IDictionary`2[System.String,Microsoft.DotNet.ImageBuilder.Model.Tag]' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.
To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.
Path 'repos[0].images[0].sharedTags', line 20, position 25.
```